services/src/main/java/org/exoplatform/task/service/TaskService.java

```diff
@@ -80,7 +80,7 @@
 
     Long countCollaboratedTasks(String user);
 
-    List<TaskDto> findTasks(String user, String query, int limit);
```

Comment:
As TaskService is an API, you should not remove this function
You should keep it, without parameter membership, and call the other by using an empty list